src/renderers/noop/ReactNoop.js

```diff
@@ -136,10 +136,22 @@ var NoopRenderer = ReactFiberReconciler({
   },
 
   scheduleAnimationCallback(callback) {
+    if (scheduledAnimationCallback) {
+      throw new Error(
+        'Scheduling an animation callback twice is excessive. ' +
+        'Instead, keep track of whether the callback has already been scheduled.'
+      );
```

Comment:
I agree. This just makes the noop renderer more restrictive so that tests fail if scheduler misbehaves. Doesn't affect public renderer API.